app/views/groupings/_group.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="one-half left team">
+  <div class="team-panel">
+    <a class="right btn btn-sm" href="#">
+      Manage
+    </a>
+    </form>
+    <span class="team-name" itemprop="name"><%= group.title %></span>
+    <p class="team-description" itemprop="description"><%= pluralize(group.repo_accesses.count, 'student') %></p>
+    <div class="team-members">
+      <% group.repo_accesses.each do |repo_access| %>
+        <% student = repo_access.user %>
+        <%= link_to student.github_user.html_url, class: 'tooltipped tooltipped-s', 'aria-label' => student.github_user.login do %>
+            <%= image_tag student.github_user.github_avatar_url(30), class: 'avatar avatar-small ', height: 30, width: 30, alt: "@#{student.github_user.login}" %>
+        <% end %>
+      <% end %>
+      <% if group.repo_accesses.count > 7 %>
+        <%= link_to '#' do %>
```

Comment:
@tarebyte Actually I wanted this link to lead the teachers to a `group#show` route, which hasn't been implemented yet.

However, since we will implement a drag-and-drop feature on this page. It will need some UI adjustment so that we can show not only 7 but all students in each group.